src/main.rs

```diff
@@ -186,11 +189,39 @@ mod tests {
     use std::sync::Mutex;
     use std::thread;
     use tempdir::TempDir;
-
     lazy_static! {
         static ref NEXT_VM_ID: Mutex<u8> = Mutex::new(1);
     }
 
+    fn get_win_size() -> (u16, u16) {
+        #[repr(C)]
+        struct WS {
+            rows: u16,
+            cols: u16,
+        };
+        let ws: WS = WS {
+            rows: 0u16,
+            cols: 0u16,
+        };
+        unsafe {
+            ioctl(0, TIOCGWINSZ, &ws);
```

Comment:
You might want to result a result of a vmm_sys_util::errno::Error generated by `Error::last()`

I think the ioctl might be failing on the Jenkins as the tests are not run inside a tty.